crates/red_knot_python_semantic/resources/mdtest/protocols.md

```diff
@@ -1215,7 +1217,7 @@ class OnlyMethodMembers(Protocol):
     def method(self) -> None: ...
 
 def f(arg1: type, arg2: type):
-    if issubclass(arg1, OnlyMethodMembers):  # error
+    if issubclass(arg1, OnlyMethodMembers):  # TODO: should emit an error here
```

Comment:
Why should this be an error? It seems to satisfy both conditions mentioned above.

Reply:
whoooops, that's a bug in the test!